.babelrc

```diff
@@ -0,0 +1,25 @@
+{
+  "presets": [
+    "react-app"
+  ],
+  "plugins": [
+    "react-hot-loader/babel",
+    "babel-plugin-transform-decorators-legacy",
+    "babel-plugin-transform-object-rest-spread",
+    "babel-plugin-syntax-trailing-function-commas",
+    "babel-plugin-transform-export-extensions",
+    [
+      "module-resolver",
+      {
+        "root": ["./src"],
+        "alias": {
```

Comment:
We can configure jest to resolve webpack alias, so there's no need to change babel configuration.

Example:
```json
"jest": {
  "moduleNameMapper": {
    "^util(.*)$": "<rootDir>/src/util$1",
    "^stores(.*)$": "<rootDir>/src/stores$1"
  }
}
```

More info https://facebook.github.io/jest/docs/webpack.html#configuring-jest-to-find-our-files